community-plugins.json

```diff
@@ -4119,5 +4119,12 @@
         "author": "musug",
         "description": "Screenshot png to jpeg and compress and rename.",
         "repo": "musug/obsidian-paste-png-to-jpeg"
+    },
+    {
+        "id": "obsidian-enhancing-export",
+        "name": "Obsidian Enhancing Export",
+        "author": "YISH<moktyish@hotmail.com>",
```

Comment:
Hi @mokeyish, just wanted to confirm: are you okay with including your email in the author section here? This will be visible from the list of community plugins, so I just wanted to confirm it's intentional

![author-section](https://user-images.githubusercontent.com/693981/167926021-830319ae-6bde-42fa-af45-0e81e46e7879.png)

Reply:
I have removed the email in it.